GAD-7.json

```diff
@@ -18,8 +25,7 @@
       "code": [
         {
           "code": "69725-0",
-          "display": "Feeling nervous, anxious or on edge",
-          "system": "http://loinc.org"
+          "display": "Feeling nervous, anxious or on edge"
```

Comment:
I'm likely remove to remove these item.code.display lines for simplicity's sake - it adds to noise here, won't be used, and we already have the references to loinc codes, where this text can be found.

Reply:
done